frontend/src/metabase-lib/viz/display.ts

```diff
@@ -76,7 +74,7 @@ export const defaultDisplay = (query: Lib.Query): DefaultDisplay => {
       return { display: "bar" };
     }
 
-    if (Lib.isCategory(column)) {
+    if (Lib.isBoolean(column) || Lib.isCategory(column)) {
```

Comment:
Introducing `Lib.isCategoryOrBoolean` (with explanatory JSDoc) could potentially help to avoid such mistakes in the future (one would type `Lib.isCat` and then they'd get a hint that there's this other function).

Maybe it'd be worth adding JSDoc to `Lib.isCategory` as well, explaining that usually it should be used together with `Lib.isBoolean` check.